src/app/Scenes/MyProfile/MyProfileSettings.tsx

```diff
@@ -41,12 +40,10 @@ export const MyProfileSettings: React.FC<MyProfileSettingsProps> = ({ onSuccess
         <Separator my={1} borderColor={separatorColor} />
         <MenuItem title="Follows" onPress={() => navigate("favorites")} />
         <Separator my={1} borderColor={separatorColor} />
-        {!!showOrderHistory && (
-          <>
-            <MenuItem title="Order History" onPress={() => navigate("/orders")} />
-            <Separator my={1} borderColor={separatorColor} />
-          </>
-        )}
+        <>
+          <MenuItem title="Order History" onPress={() => navigate("/orders")} />
+          <Separator my={1} borderColor={separatorColor} />
+        </>
```

Comment:
Maybe we could remove the `<>`? :)
```suggestion
        <MenuItem title="Order History" onPress={() => navigate("/orders")} />
        <Separator my={1} borderColor={separatorColor} />
```